packages/ui5-test-writer/templates/freestyle/webapp/test/integration/opaTests.qunit.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8" />
+	<title>Integration tests for Basic Template</title>
+
+	<script
+		id="sap-ui-bootstrap"
+		src="../../resources/sap-ui-core.js"
+		data-sap-ui-theme="<%= ui5Theme %>"
+		data-sap-ui-resourceroots='{
+			"<%= appId %>": "../../",
+			"integration": "./"
+		}'
+		data-sap-ui-animation="false"
+		data-sap-ui-compatVersion="edge"
+		data-sap-ui-async="true"
+		data-sap-ui-preload="async"
+		<% if (ui5Version === "1.71.0") { %>data-sap-ui-oninit="module:integration/opaTests.qunit"<% } %>>
+	</script>
+	<link rel="stylesheet" type="text/css" href="../../resources/sap/ui/thirdparty/qunit-2.css">
+	<script src="../../resources/sap/ui/thirdparty/qunit-2.js"></script>
+	<script src="../../resources/sap/ui/qunit/qunit-junit.js"></script>
+<% if (ui5Version !== "1.71.0") { %><script src="opaTests.qunit.js"></script><% } -%>
```

Comment:
```suggestion
<% if (templateUi5Version !== "1.71.0") { %><script src="opaTests.qunit.js"></script><% } -%>
```
I think it is better to rename this variable for clarity